src/d3a/models/strategy/update_frequency.py

```diff
@@ -156,10 +156,20 @@ def time_for_price_update(self, strategy, time_slot):
         return self.elapsed_seconds(strategy) >= \
                self.update_interval.seconds * self.update_counter[time_slot]
 
-    def update_members(self, **kwargs):
-        for key, value in kwargs.items():
-            if hasattr(self, key):
-                setattr(self, key, value)
+    def set_parameters(self, *, initial_rate_profile_buffer=None, final_rate_profile_buffer=None,
```

Comment:
Thanks a lot for this. This skipped my attention and could have lead to some issues.